workbench/src/renderer/components/SetupTab/ArgInput/index.jsx

```diff
@@ -125,174 +125,188 @@ function dragLeavingHandler(event) {
   event.currentTarget.classList.remove('input-dragging');
 }
 
-export default class ArgInput extends React.PureComponent {
-  render() {
-    const {
-      argkey,
-      argSpec,
-      userguide,
-      enabled,
-      handleBoolChange,
-      handleChange,
-      handleFocus,
-      inputDropHandler,
-      isValid,
-      selectFile,
-      touched,
-      dropdownOptions,
-      value,
-    } = this.props;
-    let { validationMessage } = this.props;
-    // Messages with this pattern include validation feedback about
-    // multiple inputs, but the whole message is repeated for each input.
-    // It's more readable if filtered on the individual input.
-    const pattern = 'Bounding boxes do not intersect';
-    if (validationMessage.startsWith(pattern)) {
-      validationMessage = filterSpatialOverlapFeedback(
-        validationMessage, value
-      );
-    }
+export default function ArgInput(props) {
```

Comment:
This diff of this function is extra "different" because the indent changed on this block. I'll drop a note where the important new bits are.